tensorboard/components/tf_ng_tensorboard/polymer_interop/polymer_interop.service.ts

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {Injectable} from '@angular/core';
+
+import {from} from 'rxjs';
+
+/** @typehack */ import * as _typeHackRxjs from 'rxjs';
+
+@Injectable()
+export class PolymerInteropService {
+  private tfStorage = document.createElement('tf-storage') as any;
+  private tfPaginatedViewStorage = (document.createElement(
+    'tf-paginated-view-store'
+  ) as any).tf_paginated_view;
+  private tfBackend = (document.createElement('tf-backend') as any).tf_backend;
+
+  fetchRuns() {
+    return from(this.tfBackend.runsStore.refresh());
+  }
+
+  fetchEnvironments() {
+    return from(this.tfBackend.environmentStore.refresh());
+  }
+
+  getPaginationLimit(): number {
+    return this.tfPaginatedViewStorage.getLimit();
+  }
```

Comment:
Angular consumers should never `getPaginationLimit()`, they should only read from ngrx state.  And I think `setPaginationLimit()` is only needed because we need Angular `changePageSize` to notify Polymer consumers.  Can we

- Drop getPaginationLimit()
- Rename setPaginationLimit() to `notifyPaginationLimitChanged()` or something?

Reply:
I will drop it for now (because it is unused). I will soon want to use it to bootstrap the initial state for it.